.github/workflows/ci-unix-static-sanitized.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-14]
```

Comment:
How about using [ubuntu-latest, macos-latest, macos-14] to make sure we do not miss when macos-latest switched to macos-15 (in a veeeery long time)

Reply:
Vincent: Thanks for the review. This pull request is not ready. I have changed it to a draft (work in progress).

I replaced macos-latest with macos-14 because I thought that from libavif's perspective, macos-latest and ubuntu-latest (both use Intel CPUs) are similar enough so that we don't need to test on both. macos-14 is documented to use the M1 CPU, so it will allow us to run our Arm code under sanitizers.

But it seems that macos-14 still uses an Intel CPU but have cross-compilers that generate Arm code. So that makes it challenging to get it to work. So I am not sure if I will be able to get this pull request to work.

Reply:
I have accepted your suggestion.